ambient.d/trains/DE_ICE.fish

```diff
@@ -1,4 +1,4 @@
-ambient_is_ssid "WIFIonICE"; or exit
+ambient_is_ssid "WIFI@DB"; or ambient_is_ssid "WIFIonICE"; or exit
```

Comment:
You can actually just pass both SSIDs as arguments and it'll return true if it's either one!

```suggestion
ambient_is_ssid "WIFI@DB" "WIFIonICE"; or exit
```

Reply:
One thing I'm curious about though: some stations also have `WiFi@DB` (eg. Friedrichstraße in Berlin), will this throw errors or return bogus data if you're not actually on a train?

Reply:
That's a good point, actually. There are also regional trains which have the same network. Since I'm currently on an ICE, I can't check that now but I'll do asap.

Reply:
Without checking in an actual station or regional train (hopefully tomorrow) I now updated the script to also check if the data return by the `status` API is actually a JSON string. So this should™ now work in other networks called the same.

Reply:
Tried in a regional train. It won't break or return Bogus data in there.